### DISCUSSION TITLE
Meshing issue using "prime.SizingType.MESHED"

### DISCUSSION THREAD

**Discussion by cadfem-flh** [ansys/pyprimemesh#731] [Q&A]
I would like to automatically mesh the remaining surfaces after I have already meshed some of them. To do this I use surface_mesh_with_size_controls().
However, if I configure the size_control object as shown:

`size_control = model.control_data.create_size_control(prime.SizingType.MESHED)
size_control.set_meshed_sizing_params(
  prime.MeshedSizingParams(model=model, growth_rate=1.2)
)
size_control.set_suggested_name("meshed_control")
size_control.set_scope(prime.ScopeDefinition(model=model))`

and then carry out the networking using the function below:

`mesh_util.surface_mesh_with_size_controls(
  size_control_names = "meshed_control",
  scope=scope,
)`

some of the zones at the edges are unnecessarily finely meshed. Does anyone already know about this "error"? How could it be fixed?

![image](https://github.com/ansys/pyprimemesh/assets/162108208/11f21195-77b4-4d71-9148-914d041341a6)

**Comment by agvarghe** [comment]
Hi @cadfem-flh .

I suspect there could be small features present at that location, I would suggest the following course of actions

1. **Inspect for Small Edges**: Examine your CAD model to identify any small edges at the problematic location.
2. **Use Space Claim**: If you find small edges, open the CAd file in Space Claim.
3. **Navigate to Repair Tab**: In the Space Claim interface, go to the **Repair** tab located in the ribbon.
4. **Select Split Edges**: Within the Repair tools, choose the **Split edges** option.
5. **Apply the Fix**: Execute the split edges operation. This should help address the issue with the small edges.

**Comment by cadfem-flh** [comment]
Hi @agvarghe .

There are no small features present at that location as you can see in the image.

![image](https://github.com/ansys/pyprimemesh/assets/162108208/13a370e6-4af3-4674-8901-ce3190fc877e)

Thank you.
Regards,
Florian
